### DISCUSSION TITLE
problems with connection docker and supabase

### DISCUSSION THREAD

**Discussion by vort2305** [supabase/supabase#12095] [Questions]
can't connect to the default project. Connection never ends...
![Supabase connection problem](https://user-images.githubusercontent.com/115911640/215990592-8969122c-fc64-4174-ad8e-8ec03e4e1bb8.png)

Could somebody help me to solve the problem.
Thanks

**Comment by shankar-narayanan-d** [comment]
Can you send the terminal for this?

**Comment by crapthings** [comment]
same here

<img width="2094" alt="image" src="https://user-images.githubusercontent.com/1147704/216095411-cc6a16c9-b762-418e-aeb3-c3365ad613d9.png">

**Comment by shankar-narayanan-d** [comment]
I think you need to update service and anon key in kong.yml file

**Comment by vort2305** [comment]
Maybe it is a stupid question but where can I find this file ( kong.yml )?

чт, 2 февр. 2023 г. в 10:16, shankar-narayanan-d ***@***.***>:

**Reply by shankar-narayanan-d** [reply]
supabase/docker/volumes/api/kong.yml
